apps/ensindexer/src/handlers/Registrar.ts

```diff
@@ -10,7 +10,7 @@ import {
   makeSubdomainNode,
 } from "@ensnode/utils";
 
-import { makeSharedEventValues, upsertAccount, upsertRegistration } from "@/lib/db-helpers";
+import { sharedEventValues, upsertAccount, upsertRegistration } from "@/lib/db-helpers";
```

Comment:
I can't see `sharedEventValues` function being called anywhere in this file. Should we drop the import, or use the function somehow?